2020/latex/ismir.sty

```diff
@@ -63,7 +63,29 @@
 %%%<<<<<<<<<<>>>>>>>>>>>%%%
 %%%<<<<<<<<<<>>>>>>>>>>>%%%
 
+\DeclareOption{lbd}{
+\def\lbd{}
+}
 
+\ProcessOptions\relax
+
+\ifdefined\lbd
+\def\papersubject{Extended Abstracts for the Late-Breaking Demo Session of the \conferenceedition\ Int. Society for Music Information Retrieval Conf., \conferenceplace, \conferenceyear}
+\else
+\def\papersubject{Proceedings of the \conferenceedition\ Int. Society for Music Information Retrieval Conf., \conferenceplace, \conferenceyear}
+\fi
+\RequirePackage{amsmath}
+\RequirePackage{cite}
+\RequirePackage{url}
+\RequirePackage{graphicx}
+\RequirePackage{color}

```

Comment:
Just to be sure, are they all required by `ismir.sty`? The ones that are not should probably stay in the `.tex` file.

Reply:
Hyperref is now required in ismir.sty to fill PDF metadata (author, title, proceedings or extended abstract in subject) towards better indexation. I got used to load hyperref after most packages to avoid clashes.